css.css

```diff
@@ -64,4 +66,49 @@ body {
 }
 
 
-/*end branch*/
\ No newline at end of file
+/*end branch*/
+
+
+/*left header*/
+
+.header .left {
+    width: 50%;
+    display: table-cell;
+}
+
+
+/*end left header*/
+
+
+/*navigation*/
+
+.navigation {
+    width: 50%;
+    height: 70px;
+    display: table-cell;
+}
+
+.navigation ul {
```

Comment:
we should be be able to use ul in css file. Please goes from the UI and analyse the name for that element and use that name in css file